tests/azure_kv_test.py

```diff
@@ -0,0 +1,95 @@
+"""Tests Azure Key Vault credential plugin."""
+
+# ruff: noqa: S106 UP045
```

Comment:
What are these? Every rule suppression must have a justification comment. Seeing alphanumeric identifiers is not enough to understand what the consequences are and what are the conditions for getting rid of temporary ignores.

Reply:
Looks like UP045 will be fixed with https://github.com/ansible/awx-plugins/pull/106/files#r2073709467 and S106 should be suppressed for all the tests and end up right after https://github.com/ansible/awx-plugins/blob/002bdc8/.ruff.toml#L155.

Reply:
Addressed in #112.